nss-day-cohort-43-csharp/Fullstack-Rage-Against-The-Virtual-Machine
# Rb users activation

Issue [27](https://github.com/nss-day-cohort-43-csharp/Fullstack-Rage-Against-The-Virtual-Machine/issues/27)

Allows Admins to deactivate profiles of other users when necessary. Deactivated profiles have restricted access to site

Testing:
- Run the new SQL scripts
- Go to Firebase and get real UIDs for user accounts in database
- Log in as admin
- Navigate to the user profiles page
- Click the deactivate button on a profile that is currently set up and cancel. Repeat and confirm
- Upon confirmation, the profile should no longer be visible.
- Try to log in as a deactivated user
- Should be notified that the account has been deactivated
- Should not be able to navigate to any pages
- Should get back unauthorized on any request made to the user profiles or category controllers. (Will need others to restrict access to other controllers)
